src/markdown/userlink/index.js

```diff
@@ -0,0 +1,3 @@
+import plugin from './remark-userlink';
```

Comment:
Absolute imports, unless the intention is to split these off into separate packages or repost.